pf/tests/internal/testprovider/testbridge.go

```diff
@@ -139,7 +139,9 @@ func (p *syntheticProvider) Metadata(_ context.Context, _ provider.MetadataReque
 func (p *syntheticProvider) Schema(_ context.Context, _ provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = pschema.Schema{
 		Attributes: map[string]pschema.Attribute{
-			"string_config_prop": pschema.StringAttribute{},
+			"string_config_prop": pschema.StringAttribute{
```

Comment:
Validators started flagging that this is not appropriate as every attribute must be Optional, or Required or Computed, or Computed Optional.